### PR TITLE
Log kube-env variables from startup script

### PR DESCRIPTION
**Pull request by mcshooter**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
To troubleshoot we'd like to have the full kube-env from the instance metadata, but it's not readily available in the startup script output. This PR will output the kube-env environments making it possible to troubleshoot based on the kube-env data.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

**Review by ibabou — APPROVED** [review]

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 1]
```diff
@@ -229,6 +229,13 @@ function Fetch-KubeEnv {
   # The type of kube_env is a powershell String.
   $kube_env = Get-InstanceMetadataAttribute 'kube-env'
   $kube_env_table = ConvertFrom_Yaml_KubeEnv ${kube_env}
+
+  Write-Host "Logging kube-env key-value pairs except CERT and KEY values"
+  foreach ($entry in $kube_env_table.GetEnumerator()) {
+    if ((-not ($entry.Name.contains("CERT"))) -and (-not ($entry.Name.contains("KEY")))) {
+      Write-Host "$($entry.Name): $($entry.Value)"
```
Comment (jeremyje) [inline cluster/gce/windows/k8s-node-setup.psm1:236]: Write_Output?

Comment (jeremyje) [inline cluster/gce/windows/k8s-node-setup.psm1:236]: Does the double reference here really work?

**Review by jeremyje — CHANGES_REQUESTED** [review]

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 1 above)
Comment (mcshooter) [reply cluster/gce/windows/k8s-node-setup.psm1:236]: haven't seen that usage yet. I have seen Write-Host and Log-Output

Comment (jeremyje) [reply cluster/gce/windows/k8s-node-setup.psm1:236]: Yes use Log-Output.

**Review by jeremyje — CHANGES_REQUESTED** [review]

**Comment by pjh** [comment]
This is great, thanks!

/lgtm
/approve

**Review by pjh — APPROVED** [review]

**Pull request merged**